### PR TITLE
Set up

### LINKED ISSUE #2: Deploy to Heroku & Integrate Travis CI

### LINKED ISSUE #3: Create PR Template

### LINKED ISSUE #11: Setup Testing Suite
- [ ] install ` pip install pytest=2.9.1` on your local
- [ ] Shaunda
- [ ] Curtis
- [ ] Jake
- [ ] Carson
- [x] George
- [ ] Olga
- [ ] Caleb

### PR DESCRIPTION
**Pull request by GeorgieGirl24**
### Description

* This sets up the Flask BE app
* Models created include:
      * User
      * Quest
      * UserQuest
      * Encounter
      * Action
* Travis CI was successfully set to deploy to Heroku
* The `manage.py` file also has the seeds for the db

### Closes issue(s)

closes #2 
closes #3 
closes #11 

### Testing

* There are no additional test in this PR

### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [ x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### Checklist
- [ ] I have written tests for this code (happy & sad)
- [ x] I have updated the Readme
- [ ] I ran full test suite - all tests passing

### Other comments
Run `dropdb database jobquest_dev;`
Run `dropdb jobquest_test;`
Run `createdb jobquest_dev;`
Run `createdb jobquest_test;`

Run `export DATABASE_URL=postgresql://localhost:5432/jobquest_dev`
Run `python manage.py db migrate`
Run `python manage.py db upgrade`

Run `export DATABASE_URL=postgresql://localhost:5432/jobquest_test`
Run `python manage.py db upgrade`

Run `python manage.py db_seed` on your local

### CODE REVIEW

File: api/database/submodels/quest.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import bleach
```
Comment (smcunning) [inline api/database/submodels/quest.py]: Just a quick comment - since we're not using the submodels right now we shouldn't merge these into main.

**Review by smcunning — APPROVED** [review]

**Review by c-bartell — APPROVED** [review]
This looks great!

**Review by carson-jardine — APPROVED** [review]
LGTM!!!

File: api/database/submodels/quest.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        """
+        db.session.commit()
+
+    def delete(self):
+        """
+        deletes model from database
+        the model must exist in the database
+        """
+        db.session.delete(self)
+        db.session.commit()
```
Comment (jakeheft) [inline api/database/submodels/quest.py]: Don't need this file

File: manage.py [hunk 3]
```diff
@@ -24,12 +24,70 @@ def db_seed():
...
+    # # quests
+    catch_troll = Quest(name='Catch a Troll for the Town Wizard', xp=200, encounter_req=1, type='active', level=1)
+    root_bandit = Quest(name='Root out the Craghook Bandits', xp=500, encounter_req=3, type='active', level=2)
+    slay_dragon = Quest(name='Slay Argoroth the Dragon', xp=1000, encounter_req=5, type='active', level=3)
+    #
+    # # user_quests
+    user_quest_1 = UserQuest(quest_id=1, user_id=1, completion_status=False, progress=1)
+    user_quest_2 = UserQuest(quest_id=2, user_id=2, completion_status=False, progress=2)
+    user_quest_3 = UserQuest(quest_id=3, user_id=3, completion_status=False, progress=1)
+    #
```
Comment (jakeheft) [inline manage.py:47]: We need to come back to these once we figure out to use the id of another variable.

File: api/database/submodels/quest.py (hunk 1 above)
Comment (GeorgieGirl24) [reply api/database/submodels/quest.py]: You got it! Done and Done!!!

**Review by jakeheft — APPROVED** [review]
Nice work getting this going!!

**Pull request merged**